File: 1-TwoSum/1-TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        int res[] = new int[2];
        for (int i = 0; i < n; i++) {
            for (int k = i+1; k < n; k++) {
                int sum = nums[i]+nums[k];
                if (sum==target) {
                    res[0]=i;
                    res[1]=k;
                    return res;
                }
            }
        }
        return res;
    }
}

File: 1-TwoSum/1-TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        hmap = {}
        for i in range(len(nums)):
            othernum = target - nums[i]
            if othernum in hmap:
                return [hmap[othernum],i]
            hmap[nums[i]] = i
